pandas/tests/indexing/multiindex/test_indexing_slow.py

```diff
@@ -7,17 +7,46 @@
 from pandas import DataFrame, Series
 import pandas._testing as tm
 
+m = 50
```

Comment:
shouldnt u do this in a function where u set the random seed? not sure it's a big deal

Reply:
we dont set randomseed in this module.  i dont think it makes  real difference

Reply:
right but this should still be in a function and just create a fixture out of it

Reply:
actually this is fine, i realize its the only thing in this file.